### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

**Review by ytsarev — CHANGES_REQUESTED** [review]
Shouldn't we put chart into /charts directory which kind of a common practice?

File: config/helm/samlet/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```
Comment (ytsarev) [inline config/helm/samlet/Chart.yaml]: Please consider syncing version and appVersion as we are doing in k8gb

**Review by ytsarev — CHANGES_REQUESTED** [review]

**Comment by k0da** [comment]
@ytsarev kubebuiler stores everything into /config, it would be logical to put chart there too. If you're strongly against. I have no problem changing it

**Comment by ytsarev** [comment]
@k0da no strong opinion that, let's keep it under `/config`

**Review by ytsarev — APPROVED** [review]

**Pull request merged**